cdap-ui/app/cdap/components/AbstractWidget/DateRangeWidget/index.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useState } from 'react';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+import { IWidgetProps } from 'components/AbstractWidget';
+import ExpandableTimeRange from 'components/TimeRangePicker/ExpandableTimeRange';
+import { DatePickerStyles } from 'components/AbstractWidget/DateTimeWidget';
+
+const styles = () => {
+  return {
+    ...DatePickerStyles(),
+  };
+};
+
+interface IDateRangeWidgetProps extends IWidgetProps<null>, WithStyles<typeof styles> {
+  format?: string; // If we want to be able to specify formats for sending to backend
```

Comment:
Can you add an example values for the format? As a user of the widget can I give any format? Would be useful if you could some example formats and point to `momentjs` documentation showing list of possible formats that can be specified.

Reply:
Also how this format passed down to time range component?

Reply:
I talked to Bhooshan earlier and we've agreed not to allow users of the widget to specify formats at this time (no pun intended). I've removed that optional prop to avoid confusion.